framework/structures.md

```diff
@@ -386,6 +390,9 @@ This structure MUST include all of the following fields:
 - [Branch Key Id](#branch-key-id)
 - [Branch Key Version](#branch-key-version)
 - [Encryption Context](#encryption-context-3)
+- [KMS ARN](#kms-arn)
+- [Create Time](#create-time)
+- [Hierarchy Version](#hierarchy-version)
```

Comment:
This is breaking. We may consider this lightly breaking because who else is constructing these. Just bringing it up.

Reply:
Fair. I think it is worth it.
I have a ticket right now where this would solve an issue.